crates/store/re_video/src/mp4.rs

```diff
@@ -94,3 +94,39 @@ pub fn load_mp4(bytes: &[u8]) -> Result<VideoData, VideoLoadError> {
         segments,
     })
 }
+
+/// Returns whether a buffer is MP4 video data.
+///
+/// From `infer` crate.
+pub fn is_mp4(buf: &[u8]) -> bool {
```

Comment:
lol, not error prone at all :see_no_evil:

Reply:
🙈

Reply:
happy to take suggestions ;-)